README.md

```diff
@@ -1,10 +1,42 @@
-# Arcaflow Engine
-
-The Arcaflow Engine allows you to run workflows using container engines, such as Docker or Kubernetes. The plugins must be built with the [Arcaflow SDK](https://arcalot.io/arcaflow/creating-plugins/python/).
-
-## Pre-built binaries
-
-If you want to use our pre-built binaries, you can find them in the [releases section](https://github.com/arcalot/arcaflow-engine/releases).
+# Arcaflow: The Noble Workflow Engine
+<img align="left" width="200px" style="padding-right: 2em; padding-bottom: 2em" alt="Arcaflow logo showing
+a waterfall and a river with 3 trees symbolizing the various plugins"
+src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
+
+Arcaflow is a highly-flexible and portable workflow system that helps you to build
+pipelines of actions via plugins. Plugin steps typically perform one action well, 
+creating or manipulating data that is returned in a machine-readable format. Data is
+validated according to schemas as it passes through the pipeline in order to clearly
+diagnose type mismatch problems early. Arcaflow runs on your laptop, a jump host, or in
+a CI system, requiring only the Arcaflow engine binary, a workflow definition in YAML,
+and a compatible container runtime.
```

Comment:
We were discussing earlier that we should not explicitly say "container", since we support non-container plugins with the correct deployer. Maybe just "runtime", or "container or language runtime".

Reply:
I considered this a bit, and right now I think the Python deployer is more of an edge use case that may be beta quality at best. Without containers, there is a lot more onus on the user to ensure dependencies are met, and there are definitely bugs and limitations to using the python deployer. Let's discuss this in Monday's community call.